File: Calculator.cpp
#include<iostream>
using namespace std;
int main(){
	int choice,a,b;
	do{
			cout<<"Enter 1 for addition"<<endl;
			cout<<"Enter 2 for Subtraction"<<endl;
			cout<<"Enter 3 for Multiplication"<<endl;
			cout<<"Enter 4 for Division"<<endl;
			cout<<"Enter 5 for Modulo"<<endl;
			cout<<endl;
			
			cout<<"Enter value of a: ";
			cin>>a;
			cout<<"Enter value of b: ";
			cin>>b;
			
			cout<<endl;
			
			cout<<"Enter your choice: ";
			cin>>choice;
			cout<<endl;
				switch(choice){
					case 1: cout<<"Addition of two no.s is: "<<a+b<<endl;
					break;
					case 2: cout<<"Subtraction of two no.s is: "<<a-b<<endl;
					break;
					case 3: cout<<"Multiplicatio of two no.s is: "<<a*b<<endl;
					break;
					case 4: cout<<"Division of two no.s is: "<<a/b<<endl;
					break;
					case 5: cout<<"Modulo of two no.s is: "<<a%b<<endl;
					break;
					case 6: cout<<"Thank you"<<endl;
					break;
					default: cout<<"Enter no.s from 1 to 5";
					break;
					}	
			}while(choice != 6);
		}

File: Calculator.php
<?php

do{
    $num1 = readline("Enter 1st no.: ");
    $num2 = readline("Enter 2nd no: ");
    echo "\n";
    echo "Enter 1 for addition".PHP_EOL;
    echo "Enter 2 for subtraction".PHP_EOL;
    echo "Enter 3 for multiplication".PHP_EOL;
    echo "Enter 4 for division".PHP_EOL;
    echo "\n";
    $choice = readline("Enter your choice: ");
    echo "\n";

    switch($choice){
        case '1':
            $sum = $num1 + $num2;
            echo "Addition of two numbers is $sum".PHP_EOL;
            echo "\n";
            break;
        case '2':
            $sub = $num1 - $num2;
            echo "Subtraction of two numbers is $sub".PHP_EOL;
            echo "\n";
            break;
        case '3':
            $mul = $num1 * $num2;
            echo "Multiplication of two numbers is $mul".PHP_EOL;
            echo "\n";
            break;
        case '4':
            $div = $num1 / $num2;
            echo "Division of two numbers is $div".PHP_EOL;
            echo "\n";
            break;
         case '5':
            echo "Thank You".PHP_EOL;
            echo "\n";
            break;
        default:
            echo "Enter correct value".PHP_EOL;
            echo "\n";
            echo "\n";
            break;    
    }

}while($choice != '5');

?>